app/grandchallenge/evaluation/models.py

```diff
@@ -1198,6 +1204,23 @@ def parent_phase_choices(self):
             )
         )
 
+    @cached_property
+    def default_interface(self):
+        try:
+            return self.algorithm_interfaces.get(
+                phasealgorithminterface__is_default=True
+            )
+        except ObjectDoesNotExist:
```

Comment:
For evaluations, we already have a property called `default_interfaces` that refers to the evaluation's inputs and outputs, not the algorithm inputs and outputs. I think it might be better to update `default_interface` to `default_algorithm_interface`. I would like to use this in the AlgorithmInterfaceForm, and so need it to be the same as the property on the algorithm. 

So I suggest to change it in both places to `default_algorithm_interface`. That property is used in multiple places (including about 50 tests), so I suggest to make the naming change in a separate PR after this one is merged.

Reply:
Removed